wandb/sdk/launch/sweeps/scheduler.py

```diff
@@ -117,10 +121,10 @@ def __init__(
         # launch agent is the one that actually runs the training workloads.
         self._workers: Dict[int, _Worker] = {}
         self._num_workers = num_workers
-        self._num_runs_launched = 0
+        self._num_runs_launched = self._kwargs.get("num_runs_launched", 0)
 
-        # Scheduler may receive additional kwargs which will be piped into the launch command
-        self._kwargs: Dict[str, Any] = kwargs
+        # Init wandb scheduler run
+        self._wandb_run = self._init_wandb_run()
```

Comment:
consider making this optional... would make tests much simpler, also could interfere with other processes in some environments (local-process?).